apisix/init.lua

```diff
@@ -313,7 +314,13 @@ end
 
 
 local function verify_tls_client(ctx)
-    if ctx and ctx.ssl_client_verified then
+    local matched = router.router_ssl.match_and_set(ctx, true)
```

Comment:
need to capture the `err` message

Reply:
match_and_set will return `err` when it fails to match the SSL certificate, but this is not considered an error here because both non-TLS and TLS will execute this function.